client/js/components/MainPageSessionCard.js

```diff
@@ -13,7 +13,13 @@ const MainPageSessionCard = () => {
           <Text style={styles.header}>HEAVY</Text>
         </TouchableOpacity>
       </View>
+<<<<<<< HEAD
```

Comment:
Oops, make sure you take care to remove conflict markers from your code before comitting.